test/protojure/plugin/core_test.clj

```diff
@@ -40,6 +40,9 @@
     (require '[com.example.addressbook :as addressbook] :reload
              '[com.example.kitchensink :as example] :reload
              '[com.example.kitchensink.Greeter.server :as example-server] :reload
+             '[com.example.enum_proto :as enum-proto]
```

Comment:
I missed this the first time, but the lack of :reload may be a problem.  TBH, I've somewhat forgotten why we needed those but I do remember it was important.

Reply:
Sounds good, added.